src/io/itk-dicom/dicom.cpp

```diff
@@ -52,6 +56,7 @@ using SeriesIndexType =
     std::unordered_map<std::string,
                        std::vector<std::string>>; // seriesUID -> []filenames
```

Comment:
`std::string` isn't just the seriesUID anymore right?

Reply:
yeah, it's our own generated/transient UID now. I'll need to update instances of seriesUID to reflect that. Maybe objectUID?